### PR TITLE
Add 'help' command

### PR DESCRIPTION
**Pull request by t-wissmann**
The 'help' command prints information on a given attribute or object.
The tests so far check that it prints something useful and matches it
against the 'doc' field in the json documentation.

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -9,6 +9,7 @@ Current git version
 
   * Client window titles (controlled by the theme attributes 'title_height',
     'title_color', 'title_font')
+  * New command 'help' for online help on the object tree
```
Comment (tiosgz) [inline NEWS:12]: This sounds like a browser will be opened with some online help :( Is it a typo of *inline*?

Comment (t-wissmann) [reply NEWS]: Hm right, maybe 'live documentation' is better (instead of 'online help').

Comment (tiosgz) [reply NEWS]: Much better, thanks!

**Review by blarz — APPROVED** [review]
Looks good to me!

**Pull request merged**